src/fparser/two/Fortran2003.py

```diff
@@ -12060,7 +12060,6 @@ def match(string):
             return Function_Name(name), Dummy_Arg_Name_List(args), Scalar_Expr(expr)
         return Function_Name(name), None, Scalar_Expr(expr)
 
-    @staticmethod
     def tostr(self):
```

Comment:
Presumably we're missing a test for this routine. Please could you add one.

Reply:
Added test file.